src/libraries/System.Runtime.InteropServices.JavaScript/System.Runtime.InteropServices.JavaScript.sln

```diff
@@ -1,4 +1,8 @@
-﻿Microsoft Visual Studio Solution File, Format Version 12.00
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 17
+VisualStudioVersion = 17.8.34004.107
```

Comment:
Are all these sln changes intentional?

Reply:
VS wants to do them any time I open the solution. So I'm thinking I should not fight it anymore.
I will drop the versions, that's not necessary.